tests/benchdnn/reduction/reduction.cpp

```diff
@@ -206,6 +207,12 @@ void skip_unimplemented_prb(const prb_t *prb, res_t *res) {
     skip_unimplemented_sum_po(prb->attr, res, dnnl_reduction, prb->sdt);
     skip_unimplemented_binary_po(prb->attr, res);
     skip_unimplemented_prelu_po(prb->attr, res, dnnl_reduction);
+#ifdef __aarch64__
+    if (is_cpu() && prb->sdt == dnnl_f32 && prb->ddt == dnnl_s8) {
+        res->state = SKIPPED;
+        res->reason = skip_reason::case_not_supported;
+    }
+#endif
```

Comment:
It should be addressed with this PR: https://github.com/uxlfoundation/oneDNN/pull/3625